snapcraft.yaml

```diff
@@ -29,6 +29,9 @@ parts:
     # FIXME: this should probably go upstream
     plugin: x-builddeb
     source: .
+    stage-packages:
+      - xdelta3
```

Comment:
Silly question, why do we need them in the snapd snap?

Reply:
We use this in case the host has no delta installed when doing delta updates.

Reply:
For the record, the docs/README coming from the packages will be gone in the snapd snap?

Reply:
this change stages the whole package instead of picking out the binaries it needs, which leaves `/usr/share/doc/` of a lot of things, as well as manpages. Compared to the whole contents of the `snap` directory (from snapd's tree) getting copied, this is minor though. Could easily be part of a separate PR that tidied things up.

Reply:
Good point, thanks, I updated this to use more parts and more targeted includes.